ee/fe/modules/500-openvpn/templates/openvpn/ingress.yaml

```diff
@@ -7,6 +7,8 @@ metadata:
   namespace: d8-{{ .Chart.Name }}
   {{- include "helm_lib_module_labels" (list . (dict "app" .Chart.Name)) | nindent 2 }}
   annotations:
+    web.deckhouse.io/export-name: "openvpn"
+    web.deckhouse.io/export-icon: "https://openvpn.net/wp-content/uploads/cropped-openvpn-32x32.png"
```

Comment:
@konstantin-axenov, we need a second opinion here. We decided to add icons for all interfaces, and it looks marvelous.
<img src="https://user-images.githubusercontent.com/32434187/164664126-18161a0b-ca94-4b18-8fc0-1ec42816e7af.png" width="250" />

However, we have a problem with icons providers.There are two options and corresponding perks:
1. We can use external links for icons. It is easy to add new interfaces, but we cannot control these icons. If someone decides to delete an icon, we will have to release a new version of Deckhouse to fix it. There is also a problem: users need access to websites with these icons (which will not work for closed environments). 

2. On the other hand, we can add all icons to the Grafana pod and serve them as static images by the Grafana server itself. This approach is solid, yet it makes adding a new interface a little more complicated.

WDYT?

Reply:
I think that the second option is the right way to do it.

Reply:
BTW, it looks really nice!

Reply:
Upmeter has its icon tho [modules/500-upmeter/images/webui/src/assets/upmeter.png](https://github.com/deckhouse/deckhouse/blob/main/modules/500-upmeter/images/webui/src/assets/upmeter.png)

Reply:
Done